bubuku/daemon.py

```diff
@@ -51,28 +50,27 @@ def main():
     kafka_properties = KafkaProperties(config.kafka_settings_template,
                                        '{}/config/server.properties'.format(config.kafka_dir))
 
-    amazon = Amazon()
-
-    address_provider = AWSExhibitorAddressProvider(amazon, config.zk_stack_name)
+    env_provider = EnvProvider.create_env_provider(config.development_mode)
```

Comment:
It makes sense to pass full `config` to `create_env_provider` call, because on next line `zk_stack` name is passed, although it may be used only in aws